ipn/ipnlocal/cert.go

```diff
@@ -87,7 +87,7 @@ var acmeDebug = envknob.RegisterBool("TS_DEBUG_ACME")
 //
 // If a cert is expired, it will be renewed synchronously otherwise it will be
 // renewed asynchronously.
-func (b *LocalBackend) GetCertPEM(ctx context.Context, domain string) (*TLSCertKeyPair, error) {
+func (b *LocalBackend) GetCertPEM(ctx context.Context, domain string, syncRenewal bool) (*TLSCertKeyPair, error) {
```

Comment:
boolean parameters aren't the prettiest at call sites, seeing mysterious "true" and "false"

Also this isn't documented.

I'd prefer a separate method than overload this. If you want to keep this, unexport it and make it an internal helper for for GetCertPEM and the new method?

Reply:
+1